File: temp/fix_all.js
const fs = require('fs');
const path = require('path');

const dir = "D:/新课开发/工作手册/高潜员工加速成长路径设计/完整课程包/02-授课PPT/slides";

const fixes = {
  'slide-26.js': [
    [/\x5c'/g, "'"],  // \' -> '
  ],
  'slide-28.js': [
    [/\x5c'/g, "'"],
  ],
  'slide-29.js': [
    [/\x5c'/g, "'"],
  ],
  'slide-38.js': [
    [/\x5c'/g, "'"],
  ],
  'slide-63.js': [
    [/'他人承担责任'/g, '"他人承担责任"'],
    [/'独立承担责任'/g, '"独立承担责任"'],
  ],
  'slide-74.js': [
    [/'绝对化表述'/g, '"绝对化表述"'],
  ],
  'slide-88.js': [
    [/"今天找你来/g, '"今天找你来'],
    [/项目进展"/g, '项目进展"'],
  ],
  'slide-91.js': [
    [/'定价'/g, '"定价"'],
  ],
  'slide-93.js': [
    [/'''/g, '"'],
  ],
  'slide-101.js': [
    [/''/g, '"'],
  ],
};

Object.entries(fixes).forEach(([file, replacements]) => {
  const fp = path.join(dir, file);
  if (!fs.existsSync(fp)) return;
  
  let content = fs.readFileSync(fp, 'utf8');
  replacements.forEach(([from, to]) => {
    content = content.replace(from, to);
  });
  fs.writeFileSync(fp, content);
  console.log('Fixed: ' + file);
});

File: temp/fix_all.py
import os
import re

slides_dir = r"D:/新课开发/企业大学/对内/1.企业大学重生：从内训中心到知识资产与智能体孵化枢纽的角色转型/PPT演示文稿/slides"

def fix_file(path):
    with open(path, 'r', encoding='utf-8') as f:
        content = f.read()

    original = content

    # Pattern 1: slide.addText(「text」, -> slide.addText("text",
    content = re.sub(r'slide\.addText\(「([^」]+)」,', r'slide.addText("\1",', content)

    # Pattern 2: slide.addText(「text」 { -> slide.addText("text", {
    content = re.sub(r'slide\.addText\(「([^」]+)」\s*\{', r'slide.addText("\1", {', content)

    # Pattern 3: slide.addText(「text with "quoted" text」 -> slide.addText("text with \"quoted\" text"
    # Handle escaped quotes inside
    content = re.sub(r'「([^」]+)"([^」]+)」', r'「\1\\"\2」', content)

    # Pattern 4: 「text「nested」text」 -> 「text【nested】text」
    content = re.sub(r'「([^「」]*)「([^」]*)」([^「」]*)」', r'「\1【\2】\3」', content)

    # Pattern 5: Fix remaining addText with corner brackets
    content = re.sub(r'slide\.addText\(「', 'slide.addText("', content)

    # Pattern 6: Fix trailing 」 after addText
    content = re.sub(r'」,\s*\{', '", {', content)
    content = re.sub(r'」,\s*\n', '",\n', content)

    # Pattern 7: Fix array items like: 「text」, -> "text",
    content = re.sub(r'「([^」]+)」,', r'"\1",', content)
    content = re.sub(r'「([^」]+)」\)', r'"\1")', content)

    if content != original:
        with open(path, 'w', encoding='utf-8') as f:
            f.write(content)
        return True
    return False

fixed = []
for f in sorted(os.listdir(slides_dir)):
    if f.startswith('slide-') and f.endswith('.js'):
        path = os.path.join(slides_dir, f)
        if fix_file(path):
            fixed.append(f)

print(f"Fixed {len(fixed)} files")
for f in fixed:
    print(f"  {f}")